build.gradle

```diff
@@ -28,5 +32,61 @@ wrapper {
 }
 
 test {
-    useJUnitPlatform()
-}
\ No newline at end of file
+    String regressionType = System.properties['regressionType']
+    systemProperty "regressionType", regressionType
+    switch (regressionType) {
+        case "@Login":
+            useJUnit()
+            include('**/LoginTest*')
+            maxParallelForks = 1
+            reports.junitXml.setEnabled(false)
+            reports.html.setEnabled(false)
+            break
+        case "@Swipe":
+            useJUnit()
+            include('**/SwipeTest*')
+            maxParallelForks = 1
+            reports.junitXml.setEnabled(false)
+            reports.html.setEnabled(false)
+            break
+        case "@Forms":
+            useJUnit()
+            include('**/FormsTest*')
+            maxParallelForks = 1
+            reports.junitXml.setEnabled(false)
+            reports.html.setEnabled(false)
+            break
+        case "@Menus":
+            useJUnit()
+            include('**/RunCukesTest*')
+            maxParallelForks = 1
+            reports.junitXml.setEnabled(false)
+            reports.html.setEnabled(false)
+            break
+        case "@All":
+            useJUnit()
+            include('**runners/*')
+            maxParallelForks = 1
+            reports.junitXml.setEnabled(false)
+            reports.html.setEnabled(false)
+            break
+    }
+
+    testLogging {
+        exceptionFormat = 'full'
+    }
+}
+
+configurations {
+    cucumberRuntime {
+        extendsFrom testImplementation
+    }
+}
+
+allure {
+    version = '2.8.1'
```

Comment:
Here you are declaring a version that does not match with the one you specified above.

Reply:
It's true ... it's my mistake, this is the allure plugin version for gradle. I make the change to the correct version.